supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/io/package.scala

```diff
@@ -29,7 +29,7 @@ package object io {
   }
 
   private [jes] def isFatalJesException(t: Throwable): Boolean = t match {
-    case e: HttpResponseException if e.getStatusCode == 403 => true
+    case e: HttpResponseException if e.getStatusCode == 403 || e.getStatusCode == 400 => true
```

Comment:
Yeah, so it seems this is the response we get back:

```json
{
  "code" : 400,
  "errors" : [ {
    "domain" : "global",
    "message" : "Pipeline 7425209317495770049: Unable to evaluate parameters: %!(EXTRA string=parameter \"infile-0\" has invalid value: FILECONTENT)",
    "reason" : "badRequest"
  } ],
  "message" : "Pipeline 7425209317495770049: Unable to evaluate parameters: %!(EXTRA string=parameter \"infile-0\" has invalid value: FILECONTENT)",
  "status" : "INVALID_ARGUMENT"
}
```

I guess I can look for `INVALID_ARGUMENT`